src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
+Spring.demo.value= test
```

Comment:
Property keys in a properties file are usually all lowercase, separated by dots. Consider renaming 'Spring.demo.value' to 'spring.demo.value'.